tests/test_evaluations/test_evaluation_functions.py

```diff
@@ -117,6 +117,31 @@ def test_evaluation_list_per_fold(self):
             self.assertIsNotNone(evaluations[run_id].value)
             self.assertIsNone(evaluations[run_id].values)
 
+    def test_evaluation_list_sort(self):
+        openml.config.server = self.test_server
```

Comment:
The test server is the default server when running unit tests, so it's unnecessary to specify.

Reply:
Actually, I think you meant to specify the production server. The test server does not have the referenced task. Because of this, no run results are retrieved, and the unit test only succeeds because empty dictionaries are retrieved.

Reply:
Oh. I see `sort_order` only works on the test server. Then please update with a task that has runs on the test server.

Reply:
Yes,  After the change from sort to sort_order, I did not check if the task exists on test server. I will add a sanity check and update this